frankenstein/chunk_store.go

```diff
@@ -121,7 +121,7 @@ func NewAWSChunkStore(cfg ChunkStoreConfig) (*AWSChunkStore, error) {
 	return &AWSChunkStore{
 		dynamodb:   dynamodb.New(session.New(dynamoDBConfig)),
 		s3:         s3.New(session.New(s3Config)),
-		memcache:   cfg.MemcacheClient,
```

Comment:
Ah, I'm going to drop the `cfg` field from the `AWSChunkStore` struct. It's better that way, since the other values in the passed-in config need to be translated/instantiated anyway and are not useful afterwards.